multiple-label-backup.md

```diff
@@ -0,0 +1,52 @@
+# Ensure support for backing up resources based on multiple labels
+
+[![hackmd-github-sync-badge](https://hackmd.io/GYmS90yOQ42GcoLaW2S4QQ/badge)](https://hackmd.io/GYmS90yOQ42GcoLaW2S4QQ)
+
+
+
+
+## Abstract
+As of today Velero supports filtering of resources based on single label per backup. It is desired that Velero support backing up of resources based on multiple labels (OR logic).
```

Comment:
NIT: `single label selector per backup`

Reply:
I would add a note that kubernetes label selectors only allow AND of labels, which is why this solution is required